test/unit/kubernetes-deploy/runner_task_test.rb

```diff
@@ -16,9 +16,9 @@ def test_run_with_invalid_configuration
     )
 
     refute task_runner.run(task_template: nil, entrypoint: nil, args: nil)
-    assert_logs_match(/Task template name can't be nil/)
-    assert_logs_match(/Namespace can't be empty/)
-    assert_logs_match(/Args can't be nil/)
+    assert_logs_match(/Task template name can't be nil/i)
+    assert_logs_match(/Namespace can't be empty/i)
+    assert_logs_match(/Args can't be nil/i)
```

Comment:
why are all those `/i` needed now?

Reply:
We changed logging which caused capitalization changes. I originally thought the diff would be easier to understand with `/i` but will flip it to just match the right capitalization